doccano_api_client/beta/client.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, base_url: str) -> None:
         headers = {
             "content-type": "application/json",
             "accept": "application/json",
+            "referer": base_url,
```

Comment:
For my own curiosity, what is the purpose of this addition? Was the API changed so this is now needed?